Snakefile

```diff
@@ -11,26 +11,29 @@ from shutil import copyfile, move
 
 from snakemake.remote.HTTP import RemoteProvider as HTTPRemoteProvider
 
-from _helpers import create_country_list, get_last_commit_message, check_config_version
+from _helpers import (
+    create_country_list,
+    get_last_commit_message,
+    check_config_version,
+    copy_default_files,
+)
 from build_demand_profiles import get_load_paths_gegis
 from retrieve_databundle_light import datafiles_retrivedatabundle
 from pathlib import Path
 
 
 HTTP = HTTPRemoteProvider()
 
-if "config" not in globals() or not config:  # skip when used as sub-workflow
-    if not exists("config.yaml"):
-        copyfile("config.tutorial.yaml", "config.yaml")
-
-    configfile: "config.yaml"
-
-
-check_config_version(config=config)
+copy_default_files()
 
 
+configfile: "config.default.yaml"
```

Comment:
Why is the default config added here? Should not be necessary in my understanding

Reply:
This is the first-order base config. With this setting the second-order configs (like config.yaml/config.tutorial.yaml) can only contain relative changes to the default config. This make config.yaml / config.tutorial.yaml etc. much shorter and easier to keep aligned with the workflow.